pokemongo_bot/cell_workers/transfer_pokemon.py

```diff
@@ -1,10 +1,10 @@
-import json
-import os
 
 from pokemongo_bot import inventory
 from pokemongo_bot.human_behaviour import action_delay
 from pokemongo_bot.base_task import BaseTask
-from pokemongo_bot.inventory import Pokemons, Pokemon, Attack
+from pokemongo_bot.inventory import Attack
+from pokemongo_bot.inventory import Pokemon
+from pokemongo_bot.inventory import Pokemons
 from operator import attrgetter
```

Comment:
Yeah, I ran `flake8` over the codebase during the conversion in e31c2a7~ -- figured it was a good time to make those changes, while I was mass-changing imports anyway. :)